packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -39,6 +39,10 @@ function createPrototype<
   const objDef = ontology.objects[type];
   const proto = {};
 
+  if (!objDef) {
```

Comment:
`undefined` and `null` are both falsey, but an object is truthy. This is a shorthand that works for these cases. You can get yourself into trouble if you check numbers like this because `1` is truthy and `0` is falsey.

![image](https://github.com/palantir/osdk-ts/assets/24275386/e7afbb56-8ba3-4fb7-bc86-2967cab70ea6)